aiolookin/protocol.py

```diff
@@ -116,8 +119,11 @@ def connection_made(self, transport: asyncio.DatagramTransport) -> None:
 
     def datagram_received(self, data: bytes, addr: Any) -> None:
         """Process incoming state changes."""
+        LOGGER.debug("Received datagram: %s", data)
         content = data.decode()
-        if not content.startswith("LOOK.in:Updated!"):
+        if not content.startswith(
```

Comment:
if not (
    content.startswith(LOOKIN_UPDATED_MESSAGE_HDR_OLD) or 
    content.startswith(LOOKIN_UPDATED_MESSAGE_HDR)
):